masterlist.yaml

```
@@ -25503,3 +25503,14 @@ plugins:
         crc: 0xB86273B2
         util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 35
+
+  - name: 'AzurasShield_SE_Eng.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12998'
+        name: 'Azura''s Shield SE'
+    dirty:
+      - <<: *quickClean
+        crc: 0x368B3AE2
+        util: '[SSEEdit v4.1.4 EXTREMELY EXPERIMENTAL](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 3
+        udr: 3
```

For cleaning reports, we prefer reports from either the latest stable or latest beta version of xEdit since those are the most likely versions users will have:

```suggestion
      - <<: *quickClean
        crc: 0x368B3AE2
        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 3
        udr: 3
```